.circleci/config.yml

```diff
@@ -1,27 +1,23 @@
-version: 2
+version: 2.1
+
+orbs:
+  catalog: giantswarm/catalog@0.0.1
+
 jobs:
-  lint-scripts:
-    docker:
-      - image: quay.io/giantswarm/shellcheck-alpine:v0.6.0
-    steps:
-      - checkout
-      - run:
-          name: lint scripts
-          command: shellcheck -x ci-scripts/*
   sync:
     machine: true
     steps:
     - checkout
 
-    - run:
-        name: Fetch new app releases and rebuild helm chart repository
-        command: ci-scripts/build-repository.sh ${CIRCLE_PROJECT_REPONAME} ${PERSONAL_ACCESS_TOKEN}
+    - catalog/sync:
```

Comment:
Using the orb, sourcecode can be found here: https://github.com/giantswarm/orbs/blob/master/catalog/orb.yaml

Reply:
How is the orb code versioned?

Reply:
When pushing to master in the `orbs` repo we automatically increase the minor version.
See here: https://github.com/giantswarm/orbs/blob/master/.circleci/config.yml#L18
Here are the docs for orb-tools https://circleci.com/orbs/registry/orb/circleci/orb-tools